moly-mofa/src/lib.rs

```diff
@@ -59,23 +59,21 @@ pub enum MofaAgent {
 }
 
 impl MofaAgent {
-    pub fn name(&self) -> String {
```

Comment:
Premature usage of `String` type here forces the use of some `.clone()`s around the code base that could be easily avoided.